### PR TITLE
Fix flaky process.cpuUsage test on linux

### PR DESCRIPTION
**Pull request by Jarred-Sumner**
### What does this PR do?

Fix flaky process.cpuUsage test on linux

### How did you verify your code works?

Tests

### CODE REVIEW

**Comment by robobun** [comment]
@Jarred-Sumner, your commit 6599bf66273070829f9bf3eac169ddc900438103 is building: [`#1769`](https://buildkite.com/bun/bun/builds/1769)

File: src/bun.js/bindings/BunProcess.cpp [hunk 1]
```diff
@@ -2294,12 +2295,18 @@ JSC_DEFINE_HOST_FUNCTION(Process_functionCpuUsage,
     if (callFrame->argumentCount() > 0) {
         JSValue comparatorValue = callFrame->argument(0);
         if (!comparatorValue.isUndefined()) {
-            if (UNLIKELY(!comparatorValue.isObject())) {
+            if (UNLIKELY(!comparatorValue.isCell())) {
```
Comment (dylan-conway) [inline src/bun.js/bindings/BunProcess.cpp]: nit: we could probably drop this check entirely, and rely on `getObject()` returning null

File: src/bun.js/bindings/BunProcess.cpp [hunk 2]
```diff
@@ -2315,17 +2322,27 @@ JSC_DEFINE_HOST_FUNCTION(Process_functionCpuUsage,
...
+                throwTypeError(globalObject, throwScope, "Expected a number for the 'system' property"_s);
                 return JSC::JSValue::encode(JSC::jsUndefined());
             }
 
-            double userComparator = userValue.asNumber();
-            double systemComparator = systemValue.asNumber();
+            double userComparator = userValue.toNumber(globalObject);
+            double systemComparator = systemValue.toNumber(globalObject);
+
+            if (userComparator > JSC::maxSafeInteger() || userComparator < 0 || std::isnan(userComparator)) {
```
Comment (dylan-conway) [inline src/bun.js/bindings/BunProcess.cpp:2331]: do we also need to check for `Infinity`?

Comment (Jarred-Sumner) [reply src/bun.js/bindings/BunProcess.cpp:2331]: infinity is greater than JSC::maxSafeInteger

**Review by dylan-conway — APPROVED** [review]

**Pull request merged**